track.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/zmb3/spotify"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+type track struct {
```

Comment:
Don't think you're using this anywhere yet right?

Reply:
the track struct?` track::getById` is being called in `main.go` for generating the playlist

Reply:
oh yeah i just realised what you meant ha

Reply:
i've removed the calling of the function but keeping the function for now in case it's needed later